.travis.yml

```diff
@@ -1,20 +1,30 @@
 language: python
 python:
-  - "3.6"
+- '3.6'
 install:
-  - "pip install pipenv"
-  - "pip install coveralls"
-  - "pipenv install"
-script:
-  coverage run manage.py test
-after_success:
-  coveralls
+- pip install pipenv
+- pip install coveralls
+- pipenv install
+script: coverage run manage.py test
+after_success: coveralls
 services:
-  - postgresql
+- postgresql
 before_script:
-  - psql -c 'create database travis_ci_test;' -U postgres
+- psql -c 'create database travis_ci_test;' -U postgres
 env:
   global:
-    - DJANGO_DB=travis_ci_test
-    - DJANGO_DB_USER=postgres
-    - DJANGO_SETTINGS_MODULE=hamlet.settings.test
+  - DJANGO_DB=travis_ci_test
+  - DJANGO_DB_USER=postgres
+  - DJANGO_SETTINGS_MODULE=hamlet.settings.test
+deploy:
+  provider: elasticbeanstalk
+  access_key_id: AKIAI4BFBKEXWULNKVGQ
+  secret_access_key:
```

Comment:
I'm guessing we don't want this to be here. Do we have a strategy for having it NOT be here?

Reply:
This is potentially okay since secret is encrypted, but I think we can encrypt both. Will double check

Reply:
Yeah FYI: https://stackoverflow.com/questions/7678835/how-secure-are-amazon-aws-access-keys

Gonna ignore encrypting access key, and will just stick to secret encryption. Will create a different user with limited rights for deploying from Travis to beanstalk